<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetrePrincipale</class>
 <widget class="QMainWindow" name="FenetrePrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rubidium</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="listeControles">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuImprimer">
     <property name="title">
      <string>Imprimer</string>
     </property>
     <addaction name="actionImprimerControlesSurveilles"/>
     <addaction name="actionToutImprimer"/>
     <addaction name="actionImprimerDetailsHeuresdesurveillance"/>
     <addaction name="actionImprimerDetailsHeuresdesurveillanceTrieParQuota"/>
    </widget>
    <addaction name="actionConnexion"/>
    <addaction name="actionDeconnexion"/>
    <addaction name="separator"/>
    <addaction name="menuImprimer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAdministration">
    <property name="title">
     <string>Administration</string>
    </property>
    <addaction name="actionAdminPoles"/>
    <addaction name="actionAdminMatieres"/>
    <addaction name="actionAdminTypesControle"/>
    <addaction name="actionAdminUtilisateurs"/>
    <addaction name="separator"/>
    <addaction name="actionAdminSalles"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuControle">
    <property name="title">
     <string>Contrôles</string>
    </property>
    <addaction name="actionNouveau_controle"/>
    <addaction name="actionNouveaux_controles"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionJeuEssai"/>
    <addaction name="actionViderBd"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAdministration"/>
   <addaction name="menuControle"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="fichierToolBar">
   <property name="windowTitle">
    <string>Fichier</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnexion"/>
   <addaction name="actionDeconnexion"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionQuitter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="controleToolBar">
   <property name="windowTitle">
    <string>Contrôles</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau_controle"/>
   <addaction name="actionNouveaux_controles"/>
  </widget>
  <widget class="QToolBar" name="toolBarActions">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSurveiller"/>
   <addaction name="actionEditer"/>
  </widget>
  <action name="actionConnexion">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/connexion</normaloff>:/icones/connexion</iconset>
   </property>
   <property name="text">
    <string>Connexion...</string>
   </property>
  </action>
  <action name="actionDeconnexion">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/deconnexion</normaloff>:/icones/deconnexion</iconset>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/quitter</normaloff>:/icones/quitter</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAdminPoles">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Pôles...</string>
   </property>
  </action>
  <action name="actionAdminMatieres">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Matières...</string>
   </property>
  </action>
  <action name="actionAdminTypesControle">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Types de contrôle...</string>
   </property>
  </action>
  <action name="actionAdminSalles">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Salles...</string>
   </property>
  </action>
  <action name="actionAdminUtilisateurs">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Utilisateurs...</string>
   </property>
  </action>
  <action name="actionNouveau_controle">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/ajouter</normaloff>:/icones/ajouter</iconset>
   </property>
   <property name="text">
    <string>Nouveau contrôle...</string>
   </property>
  </action>
  <action name="actionNouveaux_controles">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Initialisation des contrôles...</string>
   </property>
  </action>
  <action name="actionJeuEssai">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Remplir la base avec un jeu d'essai</string>
   </property>
  </action>
  <action name="actionViderBd">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/options</normaloff>:/icones/options</iconset>
   </property>
   <property name="text">
    <string>Vider la base de données</string>
   </property>
  </action>
  <action name="actionEditer">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/editer</normaloff>:/icones/editer</iconset>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
   <property name="toolTip">
    <string>Édite le controle sélectionné</string>
   </property>
  </action>
  <action name="actionSurveiller">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/surveiller</normaloff>:/icones/surveiller</iconset>
   </property>
   <property name="text">
    <string>Surveiller</string>
   </property>
   <property name="toolTip">
    <string>Surveiller ou non le contrôle courant</string>
   </property>
  </action>
  <action name="actionToutImprimer">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/imprimer</normaloff>:/icones/imprimer</iconset>
   </property>
   <property name="text">
    <string>Tous les contrôles</string>
   </property>
  </action>
  <action name="actionImprimerDetailsHeuresdesurveillance">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/imprimer</normaloff>:/icones/imprimer</iconset>
   </property>
   <property name="text">
    <string>Détails des heures de surveillance de tous les professeurs</string>
   </property>
  </action>
  <action name="actionImprimerControlesSurveilles">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/imprimer</normaloff>:/icones/imprimer</iconset>
   </property>
   <property name="text">
    <string>Contrôles surveillés</string>
   </property>
  </action>
  <action name="actionImprimerDetailsHeuresdesurveillanceTrieParQuota">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/imprimer</normaloff>:/icones/imprimer</iconset>
   </property>
   <property name="text">
    <string>Détails des heures de surveillance de tous les professeurs -- trié par quota</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>quitter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdminMatieres</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>adminMatiere()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdminPoles</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>adminPole()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdminSalles</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>adminSalles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdminTypesControle</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>adminTypeControle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdminUtilisateurs</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>adminUtilisateurs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnexion</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>seConnecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeconnexion</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>seDeconnecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNouveau_controle</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>ajouterControle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNouveaux_controles</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>initialiserControles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJeuEssai</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>remplirJeuEssai()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViderBd</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>viderBd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listeControles</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditer</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>editer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSurveiller</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>surveiller()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listeControles</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>majActionsControles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImprimerControlesSurveilles</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>imprimerListeControlesSurveilles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToutImprimer</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>imprimerListeTousControles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImprimerDetailsHeuresdesurveillance</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>imprimerListeDetailsHeuresSurveillances()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImprimerDetailsHeuresdesurveillanceTrieParQuota</sender>
   <signal>triggered()</signal>
   <receiver>FenetrePrincipale</receiver>
   <slot>imprimerListeDetailsHeuresSurveillancesTrieParQuota()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>seConnecter()</slot>
  <slot>seDeconnecter()</slot>
  <slot>adminPole()</slot>
  <slot>adminMatiere()</slot>
  <slot>adminTypeControle()</slot>
  <slot>adminUtilisateurs()</slot>
  <slot>adminSalles()</slot>
  <slot>ajouterControle()</slot>
  <slot>initialiserControles()</slot>
  <slot>quitter()</slot>
  <slot>remplirJeuEssai()</slot>
  <slot>viderBd()</slot>
  <slot>menu()</slot>
  <slot>editer()</slot>
  <slot>surveiller()</slot>
  <slot>majActionsControles()</slot>
  <slot>imprimerListeControlesSurveilles()</slot>
  <slot>imprimerListeTousControles()</slot>
  <slot>imprimerListeDetailsHeuresSurveillances()</slot>
  <slot>imprimerListeDetailsHeuresSurveillancesTrieParQuota()</slot>
 </slots>
</ui>
